app/javascript/components/users-dropdown.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div>
+    <clickable-dropdown
+      :body-title="dropdownTitle"
+      :no-items-message="noUsersMessage"
+      :items="users"
+      @item-clicked="onItemClicked"
+    /><br>
+    <div
+      v-for="(user, index) in selected"
+         :key="user.id"
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Expected indentation of 6 spaces but found 9 spaces  vue/html-indent